src/Kdyby/Doctrine/ResultSet.php

```diff
@@ -360,10 +360,10 @@ public function count()
 
 
 	/**
-	 * @param ORM\Query $query
+	 * @param ORM\AbstractQuery $query
 	 * @return ResultPaginator
 	 */
-	private function createPaginatedQuery(ORM\Query $query)
+	private function createPaginatedQuery(ORM\AbstractQuery $query)
```

Comment:
this wouldn't match the signature https://github.com/doctrine/doctrine2/blob/master/lib/Doctrine/ORM/Tools/Pagination/Paginator.php#L60